packages/react-core/src/components/Divider/__tests__/Divider.test.tsx

```diff
@@ -1,34 +1,109 @@
-import { Divider } from '../Divider';
-import { Flex, FlexItem } from '../../../layouts/Flex';
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import { Divider } from '../Divider';
+import styles from '@patternfly/react-styles/css/components/Divider/divider';
 
-test('divider using hr', () => {
-  const { asFragment } = render(<Divider />);
-  expect(asFragment()).toMatchSnapshot();
+test(`Renders with only the class name ${styles.divider} by default`, () => {
+  render(<Divider />);
+  expect(screen.getByRole('separator')).toHaveClass(styles.divider, { exact: true });
 });
 
-test('divider using li', () => {
-  const { asFragment } = render(<Divider component="li" />);
-  expect(asFragment()).toMatchSnapshot();
+test('Renders with horizontal rule by default', () => {
+  render(<Divider />);
+  expect(screen.getByRole('separator')).toContainHTML('<hr class="pf-v5-c-divider" />');
 });
 
-test('divider using div', () => {
-  const { asFragment } = render(<Divider component="div" />);
-  expect(asFragment()).toMatchSnapshot();
+test('Renders with element passed to component prop', () => {
+  render(<Divider component="li" />);
+  expect(screen.getByRole('separator')?.tagName).toBe('LI');
+});
+
+test('Test default orientation (vertical divider)', () => {
+  render(<Divider orientation={{ default: 'vertical' }} />);
+  expect(screen.getByRole('separator')).toHaveClass('pf-m-vertical');
+});
+
+test('Test sm orientation', () => {
```

Comment:
These orientation tests could be combined into a single test similar to the updated insets test and checked all at once, but not a blocker. everything else lgtm